### DISCUSSION TITLE
The command "'/usr/bin/pdftotext'

### DISCUSSION THREAD

**Discussion by andersoncarneirosilva** [spatie/pdf-to-text#65] [General]
I tried everything and I couldn't solve it.
Error:
The command "'/usr/bin/pdftotext' 'Google.pdf' '-'" failed. Exit Code: 127(Command not found) Working directory: /var/www/public Output: ================ Error Output: ================ sh: 1: exec: /usr/bin/pdftotext: not found

**Comment by FabioMezzo** [comment]
Hi! I'm getting this same error, have you solved this? I'm using it on docker

**Reply by andersoncarneirosilva** [reply]
I used the library setasign\Fpdi\Fpdi
